interface/modules/zend_modules/public/js/scripts/syndromicsurveillance.js

```diff
@@ -52,20 +52,21 @@ function clearCount()
 	return true;
 }
 
-function isNumber(evt)
+function isNumber(let evt)
```

Comment:
Unnecessary let in function parameters.

`function isNumber(evt)`

Reply:
yeah, it may be but there but other places are good enough to use let instead of var cause if we increase code further then var may occur conflicts.